packages/react-charts/src/components/ChartPoint/ChartPoint.test.tsx

```diff
@@ -2,10 +2,13 @@ import * as React from 'react';
 import { render } from '@testing-library/react';
 import { ChartLegend } from '../ChartLegend';
 import { ChartPoint } from './ChartPoint';
+import { ChartLabel } from '../ChartLabel';
 
 Object.values([true, false]).forEach(() => {
   test('ChartPoint', () => {
-    const { asFragment } = render(<ChartLegend dataComponent={<ChartPoint />} />);
+    const { asFragment } = render(
+      <ChartLegend dataComponent={<ChartPoint />} titleComponent={<ChartLabel id="test" />} />
```

Comment:
This is another case where we must clone the `titleComponent`, so we don't override the dev's given properties